website/content/docs/concepts/iam.mdx

```diff
@@ -0,0 +1,103 @@
+---
+layout: docs
+page_title: Identity and Access Management (IAM)
+description: |-
+  Identity and Access Management in Boundary
+---
+
+# Identity and Access Management (IAM)
+## IAM Components
+There are 5 key components to Boundary's identity and access management system: **Scopes**, **Auth Methods**, **Accounts**, **Users**, **Groups**, and **Roles**.
+
+#### Scopes
+[Scopes](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/scopes) are containers for all of Boundary's resources and have their own permissions 
+boundary, isolated from other scopes at the same level. **Each scope has its own set of resources and permissions**. There are three types of scopes:
+- **Global** - The single, outermost scope of Boundary that cannot be deleted.
```

Comment:
What purpose does the global scope serve?

Reply:
Added description